packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -625,102 +542,90 @@ function insertNodesToEditableTree(
 	tree: ISharedTree,
 	numberOfNodes: number,
 	shape: TreeShape,
-	dataType: string,
 ): void {
 	const treeRoot = tree.root;
 	assert(isUnwrappedNode(treeRoot));
 	let field_0;
 	let currentNode;
-	const node = testSubtrees.get(dataType);
-	assert(node !== undefined);
 	switch (shape) {
 		case TreeShape.Deep:
-			treeRoot[createField](localFieldKey, singleTextCursor(node));
+			treeRoot[createField](localFieldKey, singleTextCursor(testTreeNode));
 			assert(isUnwrappedNode(treeRoot));
 			field_0 = treeRoot[getField](localFieldKey);
 			assert(field_0 !== undefined);
 			currentNode = field_0.getNode(0);
 			for (let i = 0; i < numberOfNodes; i++) {
 				assert(isUnwrappedNode(currentNode));
-				currentNode[createField](localFieldKey, singleTextCursor(node));
+				currentNode[createField](localFieldKey, singleTextCursor(testTreeNode));
 				currentNode = currentNode[getField](localFieldKey).getNode(0);
 			}
 			break;
 		case TreeShape.Wide:
 			assert(isUnwrappedNode(treeRoot));
-			for (let i = 0; i < numberOfNodes - 1; i++) {
-				treeRoot[getField](localFieldKey).insertNodes(i, singleTextCursor(node));
+			for (let i = 0; i < numberOfNodes; i++) {
+				treeRoot[getField](localFieldKey).insertNodes(i, singleTextCursor(testTreeNode));
 			}
 			break;
 		default:
 			unreachableCase(shape);
 	}
 }
 
-function getTestTreeAsJSObject(
-	numberOfNodes: number,
-	shape: TreeShape,
-	dataType: string,
-): Jsonable {
+function getTestTreeAsJSObject(numberOfNodes: number, shape: TreeShape): Jsonable {
```

Comment:
Most (if not all) users of this function just have a constant for the shape. This I think it would be simpler for the users of this function to just use getJSTestTreeDeep or getJSTestTreeWide directly, and remove this function if that leaves no users.

Reply:
removed unnecessary `getTestTreeAsJSObject` function